docs/user/general/datatypes.rst

```diff
@@ -320,7 +319,7 @@ Conversion from DATE
 Conversion from TIME
 >>>>>>>>>>>>>>>>>>>>
 
-- Time value cannot be converted to any other date and time types since it does not contain any date information, so it is not meaningful to give no date info to a date/datetime/timestamp instance.
+- When time value is converted to any other datetime types, the date part of the new value is filled up with today's date. For example, (like `CURDATE` function); a time value X converted to a timestamp would produce today's date at time X.
```

Comment:
```suggestion
- When time value is converted to any other datetime types, the date part of the new value is filled up with today's date, like with the `CURDATE` function. For example, a time value X converted to a timestamp would produce today's date at time X.
```

I think the grammar here might be a little bit off, the placement of the parentheses were a tiny bit off and I don't believe a semicolon should be placed as it it was.